pages/search.vue

```diff
@@ -149,7 +159,7 @@
  };
  */
 
- let showContactDetails = (contact) => {
+ let showContactDetails = (contact: any) => {
```

Comment:
This is not proper usage of refs. Refs should always be const, and you set their value via `ref.value`